netmgrd.te

```diff
@@ -26,33 +26,29 @@ unix_socket_connect(netmgrd, qmuxd, qmuxd)
 
 allow netmgrd sysfs_net:dir r_dir_perms;
 allow netmgrd sysfs_net:file rw_file_perms;
-allow netmgrd sysfs_soc:dir search;
-allow netmgrd sysfs_soc:file r_file_perms;
-allow netmgrd sysfs_msm_subsys:dir r_dir_perms;
-allow netmgrd sysfs_msm_subsys:file r_file_perms;
-
-allow netmgrd system_file:file lock;
 
 r_dir_file(netmgrd, sysfs_msm_subsys)
+r_dir_file(netmgrd, sysfs_soc)
 
 wakelock_use(netmgrd)
 
-#Ignore if device loading for private IOCTL failed
-dontaudit netmgrd kernel:system { module_request };
+# Allow netutils usage
+domain_auto_trans(netmgrd, netutils_wrapper_exec, netutils_wrapper)
 
 allow netmgrd proc_net:file rw_file_perms;
 allow netmgrd netmgr_data_file:dir rw_dir_perms;
 allow netmgrd netmgr_data_file:file create_file_perms;
-allow netmgrd system_file:file execute_no_trans;
+allow netmgrd system_file:file { execute_no_trans lock };
```

Comment:
Do that to match CAF:
```
allow netmgrd system_file:file x_file_perms;
not_full_treble(`allow netmgrd system_file:file lock;')
```

Reply:
Ok thanks I will update soon .... Without this PR RIL is unknown for Loire on Enforcing ....

Reply:
How is this possible? netmgrd is only used for mobile data.

Reply:
Is possible ... If you want I can post screens xD before - after

Reply:
I got ril on 8.1 and enforcing here on kugo,without this pr

Reply:
6.4 kernel @oshmoun ?